typings/index.d.ts

```diff
@@ -353,7 +369,7 @@ declare module 'enmap' {
      * points.inc("numberInObject", "sub.anInt"); // {sub: { anInt: 6 }}
      * @returns The enmap.
      */
-    public inc(key: K, path?: string): this;
+    public inc(key: string, path?: string): this;
```

Comment:
What happened to string[]?

Reply:
string[] was never a part of the function and is not supported by it.